pkg/dispatcher/dispatcher.go

```diff
@@ -196,12 +229,6 @@ func (d *Dispatcher) dispatch(ctx context.Context, msg amqp.Delivery, ceClient c
 
 	response, result := ceClient.Request(ctx, *event)
 	statusCode, isSuccess := getStatus(ctx, result)
-	if statusCode != -1 {
-		args := &dispatcher.ReportArgs{EventType: event.Type()}
-		if err = d.Reporter.ReportEventCount(args, statusCode); err != nil {
```

Comment:
Did we stop reporting counts of errors?  That feels like a useful metric to keep.

It's also possible the defer is getting the http error somehow -- if so, that's probably a detail worth commenting, since earlier code is depending on the actions of later code, rather than the usual.